### PR TITLE
Save validated simulation data to BE

### PR DESCRIPTION
**Pull request by jnwltr**
- plus add essential test based on cz real scenario

### CODE REVIEW

File: backend/src/game/routes.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import express from 'express';
+import {GameDataModel} from './model';
+import {validateGame} from '../../../frontend/src/app/services/validate';
+
+
+export const router = express.Router();
+
+// router.get('/api/todo', async (req, res) => {
```
Comment (JaroslavSoldat) [inline backend/src/game/routes.ts:8]: Consider add TODO or remove it completely.

File: frontend/README.md [hunk 2]
```diff
@@ -1,14 +1,5 @@
...
-
-### Install
-- `cd frontend`
-- `npm i`
-
-### Run
-- `npm start` for a dev server at `http://localhost:4200/`
-
 ## Code scaffolding
 Run `npx ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
```
Comment (JaroslavSoldat) [inline frontend/README.md:5]: What about this? Consider removing it as well.

**Review by JaroslavSoldat — APPROVED** [review]

File: frontend/README.md (hunk 2 above)
Comment (jnwltr) [reply frontend/README.md:5]: Feel free to do it yourself.

File: backend/src/game/routes.ts (hunk 1 above)
Comment (jnwltr) [reply backend/src/game/routes.ts:8]: It's a clear todo I will work on today.

**Pull request merged**